### ISSUE TITLE
Review of Quantitative Empirical Evaluations of Technology for People with Visual Impairments

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#164]
### Links
- Paper: https://hal.archives-ouvertes.fr/hal-02437881/document

### Abstract
top-tier（CHI, ASSETS, TOCHI, TACCESS）に採択された視覚障害者支援の論文243本を対象に、どのようにユーザ実験を行なっているか調査した論文

参加人数、評価方法（パラメタや評価尺度、検定の種類など）、実験場所（ラボ内か実環境か）等々のデータ分析に加え、実験で大変だった点（各論文の著者が論文内で報告しているもの）がまとめられている

結果をレポートするだけでなく、最後により良いユーザ実験を行うためのアドバイスを提案してくれている

**Comment by dayvsonsales** [comment]
Hi, I liked your repository. It helps a lot. But I want to make a suggestion, you should translate it to English in order to help more people to find it and use it as a reference.

**Comment by Wotipati** [comment]
@dayvsonsales 

Thanks for your comment!!

Sorry, I have no plan to translate contents.

If I copy and paste the abstract of the paper to each issue, is it helpful for you and others?
For example, we can search papers by using some keywords.

Best,

**Comment by dayvsonsales** [comment]
Yes, It will be. Putting a least the paper abstract will help more people find your repository in searches engine like Google and understand better.

Keep going, you're doing a great work selecting those papers 👍🏻

**Comment by Wotipati** [comment]
@dayvsonsales 
I added the abstracts in all issues!!

Please share this repository if you like it!!

This rink is my tweet!!
https://twitter.com/_wotipati_/status/1235099878365073408?s=20

Best,
